zhaquirks/senoro/senorowin.py

```diff
@@ -0,0 +1,43 @@
+"""Sensoro Window Sensor (TS0601)."""
+
+from zigpy.quirks.v2.homeassistant import EntityPlatform, EntityType
+import zigpy.types as t
+
+from zhaquirks.tuya import BatterySize
+from zhaquirks.tuya.builder import TuyaQuirkBuilder
+
+
+class OpeningStateEnum(t.enum8):
+    """Enum for opening state."""
+
+    open = 0
+    closed = 1
+    tilted = 2
+
+
+(
+    TuyaQuirkBuilder("_TZE200_ytx9fudw", "TS0601")
+    .tuya_battery(
+        dp_id=2,
+        battery_type=BatterySize.CR2032,
+        battery_qty=3,
+    )
+    .tuya_enum(
+        dp_id=101,
+        attribute_name="opening_state",
+        enum_class=OpeningStateEnum,
+        translation_key="opening_state",
+        fallback_name="Opening state",
```

Comment:
Let's move `translation_key` and `fallback_name` parameters to the bottom, to follow the style with how other quirks v2 entities are defined.